scripts/GetCardImages.js

```diff
@@ -0,0 +1,37 @@
+const axios = require('axios');
+const fs = require('fs');
+const path = require('path');
+const utils = require('../src/cards/utils');
+const chaseFuncs = require('../src/cards/chase');
+const CHASE_DOMAIN = "https://creditcards.chase.com";
+
+async function main() {
+    let chase = new chaseFuncs();
+    let cards = await chase.getBody();
+
+    for (let card of cards) {
+        downloadImage(card.name, card.link);
+    }
+}
+
+async function downloadImage(title, imgPath) {
+    title = utils.lettersOnly(title).toLowerCase();
+    const url = `${CHASE_DOMAIN}${imgPath}`;
+    const writePath = path.resolve(__dirname, '../images', `${title}.png`);
+    const writer = fs.createWriteStream(writePath);
+
+    const response = await axios({
+        url,
+        method: 'GET',
+        responseType: 'stream'
+    });
+
+    response.data.pipe(writer);
+
+    return new Promise((resolve, reject) => {
+        writer.on('finish', resolve);
+        writer.on('error', reject);
+    })
+}
+
+main();
```

Comment:
Should this script be automatically triggered somehow/under certain circumstances?

Reply:
Hmmm...so i want to auto trigger it, but haven't thought of how within the server itself since it's not awake 24/7. I'm using heroku as my backend server and it only wakes up when someone is using it...lol...so i just have it as manual rn.

Reply:
So there is a backend piece connected and utilizing this library?

Reply:
Correct.